db/seeds.rb

```diff
@@ -115,3 +115,5 @@
 )
 
 User.create!(email: 'admin@example.com', first_name: 'Bossy', last_name: 'McBosserton', password: 'password', role: :admin)
+User.create!(email: 'cj.kelling88@gmail.com', first_name: 'C', last_name: 'McBoss', password: 'password', role: :default)
+User.create!(email: 'macoop2363@gmail.com', first_name: 'M', last_name: 'McBoss', password: 'password', role: :default)
```

Comment:
Probably not wise to store our emails in a seed file, but they are both public on github, so...